### DISCUSSION TITLE
Server Verifiction Error

### DISCUSSION THREAD

**Discussion by PointZeroSeven** [mastodon/mastodon#20204] [General]
I get this error message every time I try to log in.

**Comment by Gargron** [comment]
Please make sure cookies are enabled.

**Reply by flyingfrawg** [reply]
Hunted your tip down and I am confused by the message and by your answer. I don't see a cookies setting for the iOS Mastodon App anywhere accessible from the app or in the iOS settings panel. Is this a different App cookies setting like a browser? I have no problem logging in from a browser at all, either on the desktop or in Firefox for iOS. Thanks! I've experienced the same thing with Metatext on iOS as well...
